packages/react/src/components/masthead.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { PropsWithChildren, useEffect } from "react";
+import { ChevronDownIcon } from "../icons/chevron-down";
+import { Lock2Icon } from "../icons/lock-2";
+import SolidLockIcon from "../icons/solid-lock";
+import MalaysiaFlagIcon from "../icons/malaysia-flag";
+import { GovMyIcon } from "../icons/gov-my";
+
+// Types
+type MastheadContentProps = {
+  icon: React.ReactNode;
+  children: React.ReactNode;
+};
+
+type MastheadSectionTitleProps = {
+  children: React.ReactNode;
+};
+
+type MastheadSectionBodyProps = {
+  children: React.ReactNode;
+};
+
+// Component definitions
+export const MastheadHeader = ({ children }: PropsWithChildren) => {
+  return (
+    <summary className="block cursor-pointer list-none py-2.5 outline-none sm:py-1 [&::-webkit-details-marker]:hidden">
+      <div className="px-4.5 mx-auto flex max-w-[1280px] items-center gap-1.5 text-sm/4 max-sm:justify-between md:px-6">
+        {children}
+      </div>
+    </summary>
+  );
+};
+
+export const MastheadOfficialIndicator = ({
+  children = "Official Malaysia Government Website",
+}: PropsWithChildren) => {
+  return (
+    <div className="flex select-none items-center gap-2">
+      <MalaysiaFlagIcon className="shrink-0" />
+      <span className="text-txt-black-900">{children}</span>
+    </div>
+  );
+};
+
+export const MastheadIdentificationToggle = ({
```

Comment:
Rename `MastheadToggle`

Reply:
Does not highlight when tabbed (in focus)

Reply:
fixed